src/caret_analyze/plot/visualize_lib/bokeh/util/hover.py

```diff
@@ -138,18 +153,123 @@ def create(self, options: dict = {}) -> HoverTool:
         )
 
 
+class CallbackSchedBarKeys(HoverKeysBase):
+
+    def __init__(self, target_object: TargetTypes) -> None:
+        super().__init__(target_object)
+
+    def _validate(self, target_object: Any) -> None:
+        if not isinstance(target_object, CallbackBase):
+            raise InvalidArgumentError(
+                "'target_object' must be CallbackBase in callback scheduling graph."
+            )
+
+    def to_list(self) -> List[str]:
+        return ['legend_label', 'node_name', 'callback_name',
+                'callback_type', 'callback_param', 'symbol']
+
+
+class CallbackSchedRectKeys(HoverKeysBase):
+
+    def __init__(self, target_object: TargetTypes) -> None:
+        super().__init__(target_object)
+
+    def _validate(self, target_object: Any) -> None:
+        if not isinstance(target_object, CallbackBase):
+            raise InvalidArgumentError(
+                "'target_object' must be CallbackBase in callback scheduling graph."
+            )
+
+    def to_list(self) -> List[str]:
+        return ['legend_label', 'callback_start', 'callback_end', 'latency']
+
+
+class TimeSeriesKeys(HoverKeysBase):
+
+    def __init__(self, target_object: TargetTypes) -> None:
+        super().__init__(target_object)
+
+    def _validate(self, target_object: Any) -> None:
+        if not isinstance(target_object, (CallbackBase, Communication, Publisher, Subscription)):
+            raise InvalidArgumentError(
+                "'target_object' must be [CallbackBase/Communication/Publisher/Subscription]"
+                'in timeseries graph.'
+            )
+
+    def to_list(self) -> List[str]:
+        if isinstance(self._target_object, CallbackBase):
```

Comment:
```suggestion
    def to_list(self) -> List[str]:
        hover_keys: List[str] = []
        if isinstance(self._target_object, CallbackBase):
```

Reply:
Fixed in https://github.com/tier4/CARET_analyze/pull/277/commits/3082601e1601048319240a444cf61997f1a65bf3